.travis.yml

```diff
@@ -4,6 +4,9 @@ cache: bundler
 rvm:
 - 2.4
 - 2.0
+env:
+  - CHECK = spec
+  - CHECK = license_manager
```

Comment:
This should be `license_finder`

Reply:
And I think this should just be added to the `bundle exec rake travisci rubocop` invocation. Currently it's trying to run the check from within the `bolt-modules/boltlib` directory rather than the top-level, and it's also running all the other tests before doing the license check (while also running them all separately).